[2024-03-21T19:51:46.684-0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_multiple_tasks3.task_G manual__2024-03-21T22:51:32.407806+00:00 [queued]>
[2024-03-21T19:51:46.688-0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_multiple_tasks3.task_G manual__2024-03-21T22:51:32.407806+00:00 [queued]>
[2024-03-21T19:51:46.688-0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-03-21T19:51:46.688-0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2024-03-21T19:51:46.688-0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-03-21T19:51:46.702-0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): task_G> on 2024-03-21 22:51:32.407806+00:00
[2024-03-21T19:51:46.703-0300] {standard_task_runner.py:55} INFO - Started process 3676 to run task
[2024-03-21T19:51:46.705-0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'executing_multiple_tasks3', 'task_G', 'manual__2024-03-21T22:51:32.407806+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/multiple_tasks3.py', '--cfg-path', '/tmp/tmpgo_p22e4']
[2024-03-21T19:51:46.706-0300] {standard_task_runner.py:83} INFO - Job 54: Subtask task_G
[2024-03-21T19:51:46.736-0300] {task_command.py:389} INFO - Running <TaskInstance: executing_multiple_tasks3.task_G manual__2024-03-21T22:51:32.407806+00:00 [running]> on host DESKTOP-0DNP0LN.localdomain
[2024-03-21T19:51:46.774-0300] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=falcone
AIRFLOW_CTX_DAG_ID=executing_multiple_tasks3
AIRFLOW_CTX_TASK_ID=task_G
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T22:51:32.407806+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-21T22:51:32.407806+00:00
[2024-03-21T19:51:46.775-0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-03-21T19:51:46.775-0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Task G completed!']
[2024-03-21T19:51:46.778-0300] {subprocess.py:86} INFO - Output:
[2024-03-21T19:51:46.778-0300] {subprocess.py:93} INFO - Task G completed!
[2024-03-21T19:51:46.778-0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-21T19:51:46.792-0300] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=executing_multiple_tasks3, task_id=task_G, execution_date=20240321T225132, start_date=20240321T225146, end_date=20240321T225146
[2024-03-21T19:51:46.837-0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-03-21T19:51:46.845-0300] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
